### ISSUE TITLE
I assume it currently not supports cds version 6.1

### ISSUE THREAD

**Issue by mhay-at-dvelop** [sapmentors/cds-pg#343]
Hi,
upgraded to cds 6.1.2 in my cap-app.
this resulted in error at the very first database statement:

**SCRAM-SERVER-FIRST-MESSAGE: client password must be a string**

After going back to 5.9.8 my cap app worked again.

kind regards
Matthias

**Comment by vobu** [comment]
we're not on `cds^6` yet with `cds-pg` - but please, don't hesitate and bolt it in via a PR! 👍

**Comment by ryegrosT8** [comment]
Hi there, 

I would like to contribute here. I took a look at source code in @sap/cds 6.5.0 and seems that two changes are affecting this project:

1 - The source at @sap/cds/lib/deploy now is available at @sap/cds/lib/dbs/cds-deploy, needs to fix references on cds-pg.
2 - Main service class needs to implement getDbUrl method providing database connection string (seems to not be used in cds-pg case, but needs to be implemented)

I did the changes at my fork and will submit it as PR ( PR: https://github.com/sapmentors/cds-pg/pull/383 ), the main test's classes are already fixed and working, I don't know if you guys want to run some more additional tests.

Ps. Working on  lts/fermium -> v14.20.0 and lts/gallium -> v16.16.0 .
